odoo/addons/base/data/res_currency_data.xml

```diff
@@ -1687,34 +1607,146 @@
+            <field name="name">BOV</field>
+            <field name="full_name">Bolivian Mvdol (funds code)</field>
+            <field name="symbol">$Bs</field>
             <field name="rounding">0.01</field>
             <field name="active" eval="False"/>
-            <field name="currency_unit_label">Dinar</field>
-            <field name="currency_subunit_label">Para</field>
+            <field name="currency_unit_label">Bolivian boliviano</field>
+            <field name="currency_subunit_label">centavo</field>
         </record>
 
-        <record id="GTQ" model="res.currency">
-            <field name="name">GTQ</field>
-            <field name="full_name">Guatemalan quetzal</field>
-            <field name="symbol">Q</field>
+        <record id="CUC" model="res.currency">
+            <field name="name">CUC</field>
+            <field name="full_name">Cuban convertible peso</field>
+            <field name="symbol">$</field>
             <field name="rounding">0.01</field>
             <field name="active" eval="False"/>
-            <field name="currency_unit_label">Quetzal</field>
-            <field name="currency_subunit_label">Centavo</field>
+            <field name="currency_unit_label">Cuban convertible peso</field>
+        </record>
+
+        <record id="VES" model="res.currency">
+            <field name="name">VES</field>
+            <field name="full_name">Venezuelan bolívar soberano</field>
+            <field name="symbol">Bs</field>
+            <field name="rounding">0.01</field>
+            <field name="active" eval="False"/>
+        </record>
+
+        <record id="UYW" model="res.currency">
+            <field name="name">UYW</field>
+            <field name="full_name">Unidad previsional</field>
+            <field name="symbol">$</field>
+            <field name="rounding">0.0001</field>
+            <field name="active" eval="False"/>
+            <field name="currency_unit_label">peso</field>
+            <field name="currency_subunit_label">centésimo</field>
+        </record>
+
+        <record id="UYI" model="res.currency">
+            <field name="name">UYI</field>
+            <field name="full_name">Uruguay Peso en Unidades Indexadas</field>
+            <field name="symbol">$</field>
+            <field name="rounding">1</field>
+            <field name="active" eval="False"/>
+            <field name="currency_unit_label">Peso</field>
+            <field name="currency_subunit_label">centésimo</field>
+        </record>
+
+        <record id="USN" model="res.currency">
+            <field name="name">USN</field>
+            <field name="full_name">United States dollar (next day)</field>
+            <field name="symbol">$</field>
+            <field name="rounding">0.1</field>
+            <field name="active" eval="False"/>
+            <field name="currency_unit_label">Dollars</field>
+            <field name="currency_subunit_label">Cents</field>
+        </record>
```

Comment:
As I understand it, it's a particular case of the USD. Probably not worth adding.